Makefile

```diff
@@ -113,12 +118,26 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/crio
+	rm -f bin/crio.cross.*
 	$(MAKE) -C conmon clean
 	$(MAKE) -C pause clean
 	rm -f test/bin2img/bin2img
 	rm -f test/copyimg/copyimg
 	rm -f test/checkseccomp/checkseccomp
 
+# the approach here, rather than this target depending on the build targets
+# directly, is such that each target should try to build regardless if it
+# fails. And return a non-zero exit if _any_ target fails.
+local-cross:
+	@$(MAKE) --keep-going $(CROSS_BUILD_TARGETS)
+
+bin/crio.cross.%:
```

Comment:
Oops, hit approve too soon ;).  I [still][1] think we need `.gopathok` here, since `.gopathok` ensures you have a Go environment that can find the local source directory.  That means any rules building local Go source (including this one) would need it as a dependency.

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1506#discussion_r193137696

Reply:
k